### ISSUE TITLE
Form Fields Error (tag-input, input)

### ISSUE THREAD

**Issue by elvisgn** [creativetimofficial/material-dashboard-pro-angular#116]
Hi
Having an issue with the tag-input mapped to an input array (please see attached gif). On deleting a tag, should I be calling a method to remove the field from the array, or would it happen automatically ?
![tag-input](https://user-images.githubusercontent.com/8165850/31763600-cb98831e-b4cf-11e7-888b-476b77d1d14a.gif)

Also, notice the floating title of the text area field overlapping the content.

Following is my code
```
<form #f="ngForm" novalidate (ngSubmit)="save(f.value, f.valid)">

	<div class="card-header card-header-icon" data-background-color="rose">
		<i class="material-icons">assignment</i>
	</div>

	<div class="card-content">

		<h4 class="card-title">Event Details</h4>

		<div class="form-group label-floating">
			<label class="control-label" for="eventTitle">Event Title
				<span class="star">*</span>
			</label>
			<input mdInput type="text" class="form-control" name="eventTitle" [(ngModel)]="conference.title"
				required #eventTitle="ngModel" pattern="^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+$">
			<small [hidden]="conference.title.valid || (conference.title.pristine && !f.submitted)" class="text-danger">
				Event title is required.
			</small>
		</div>

		<div class="form-group label-floating">
			<label class="control-label" for="eventDescription">Event Description
				<span class="star">*</span>
			</label>
			<textarea #eventDescription class="form-control" name="eventDescription" [(ngModel)]="conference.description" md-maxlength="150" rows="5" pattern="^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+$"></textarea>
			<small [hidden]="conference.description.valid || (conference.description.pristine && !f.submitted)" class="text-danger">
				Event description is required.
			</small>
		</div>

		<div class="row">
			<div class="col-md-12">
				<div class="form-group label-floating">
					<label class="control-label" for="tracks">Tracks</label>
					<tag-input #tracks name="tracks" [(ngModel)]='conference.tracks' theme='filled-theme'></tag-input>
				</div>
			</div>
		</div>
		
		<div class="category form-category">
			<span class="star">*</span> Required fields
		</div>

		<div class="form-footer text-right">

			<div class="form-group">
				<button type="submit" class="btn btn-rose btn-fill btn-wd" [disabled]="!f.valid">Create / Update</button>
			</div>

		</div>
	</div>
</form>
```

**Comment by elvisgn** [comment]
I think the problem seems to be using the material and form controls in the same page. Even the documentation seems to be all mixed up with suggestions to use mat, md and form-controls.

Can you kindly provide a single form that covers all the possible UI elements...either only using bootstrap + angular form, or using material only.

**Comment by elvisgn** [comment]
Can someone help...can't get a simple form to display elegantly...can't understand what the classes are doing to the UI..introducing ngModel causes weird reactions.

I just need this json data to be captured from a form...it feels like the elements can't be used with one another.
```
{
  "title": "Dummy Title 2017",,
  "description": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut",
  "endDate": "2017-03-20T20:41:38.170Z",
  "location": "Dummy location",
  "startDate": "2017-03-24T20:41:38.170Z",
  "timeEnd": "5:00 pm",
  "timeStart": "8:00 am",
  "tracks": [
        "Ionic",
        "BPM",
        "Testing",
        "Tooling",
        "IoT",
        "Blockchain"
    ]
}
```

**Comment by chelaruc** [comment]
Hi, @elvisgn! Thank you for using our product! We will do an update this week with new forms validation.

Best,
Ciprian

**Comment by chelaruc** [comment]
fixed in v1.4.5

**Comment by alkheyal** [comment]
how can i use this material-dashboard-pro-angular in my ionic app i created it before
